### ISSUE TITLE
Update bbi flags

### ISSUE THREAD

**Issue by seth127** [metrumresearchgroup/rbabylon#123]
There are flags that can be passed to bbi through `.bbi_args` and these are checked against [a constant defined in aaa.R](https://github.com/metrumresearchgroup/rbabylon/blob/master/R/aaa.R#L4) to make sure they exist. These have gotten out-of-date so they need to be updated to reflect the current flags available in `bbi nonmem --help`.

**Comment by seth127** [comment]
Verified these are the flags as of the `v2.3.0-rc1` commit:
```
✔ ~/go/src/github.com/metrumresearchgroup/babylon [develop|✔] 
15:21 $ git rev-parse HEAD
d15ef96c4926a84169632a6f37ece32cbaf883a1
✔ ~/go/src/github.com/metrumresearchgroup/babylon [develop|✔] 
15:31 $ go build -o /usr/local/bin/bbi cmd/bbi/main.go 
✔ ~/go/src/github.com/metrumresearchgroup/babylon [develop|✔] 
15:32 $ /usr/local/bin/bbi nonmem --help

run nonmem model(s), for example: 
bbi nonmem run <local|sge> run001.mod
bbi nonmem run  --clean_lvl=1 <local|sge> run001.mod run002.mod
bbi nonmem run <local|sge> run[001:006].mod // expand to run001.mod run002.mod ... run006.mod local
bbi nonmem run <local|sge> .// run all models in directory
 

summarize model(s), for example: 
bbi nonmem summary run001/run001
bbi nonmem summary run001/run001.lst
bbi nonmem summary run001/run001.res
 

load .cov and .cor output from model(s), for example: 
bbi nonmem covcor run001/run001
bbi nonmem covcor run001/run001.cov

Usage:
  bbi nonmem [flags]
  bbi nonmem [command]

Available Commands:
  clean       clean files and folders
  covcor      load .cov and .cor output from a model run
  probs       summarize information about project
  reclean     clean files in an estimation directory by clean level
  run         run a (set of) models locally or on the grid
  scaffold    scaffold directory structures
  summary     summarize the output of model(s)

Flags:
      --background             RAW NMFE OPTION - Tells nonmem not to scan StdIn for control characters
  -h, --help                   help for nonmem
      --licfile string         RAW NMFE OPTION - Specify a license file to use with NMFE (Nonmem)
      --maxlim int             RAW NMFE OPTION - Set the maximum values set for the buffers used by Nonmem (default 100)
      --mpi_exec_path string   The fully qualified path to mpiexec. Used for nonmem parallel operations (default "/usr/local/mpich3/bin/mpiexec")
      --nm_version string      Version of nonmem from the configuration list to use
      --nmqual                 Whether or not to execute with nmqual (autolog.pl
      --nobuild                RAW NMFE OPTION - Skips recompiling and rebuilding on nonmem executable
      --parafile string        Location of a user-provided parafile to use for parallel execution
      --parallel               Whether or not to run nonmem in parallel mode
      --parallel_timeout int   The amount of time to wait for parallel operations in nonmem before timing out (default 2147483647)
      --prcompile              RAW NMFE OPTION - Forces PREDPP compilation
      --prsame                 RAW NMFE OPTION - Indicates to nonmem that the PREDPP compilation step should be skipped

Global Flags:
  -d, --debug         debug mode
      --json          json tree of output, if possible
  -p, --preview       preview action, but don't actually run command
      --threads int   number of threads to execute with locally or nodes to execute on in parallel (default 4)
  -v, --verbose       verbose output

Use "bbi nonmem [command] --help" for more information about a command.
```

**Comment by seth127** [comment]
Here's some R code to show how I generated the list to update, since this is all ad hoc:
```
f1 <- c(
  "--background",
  "--help",
  "--licfile",
  "--maxlim",
  "--mpi_exec_path",
  "--nm_version",
  "--nmqual",
  "--nobuild",
  "--parafile",
  "--parallel",
  "--parallel_timeout",
  "--prcompile",
  "--prsame",
  "--debug",
  "--json",
  "--preview",
  "--threads",
  "--verbose"
)

f2 <- c(
  "--additional_post_work_envs",
  "--clean_lvl",
  "--config",
  "--copy_lvl",
  "--delay",
  "--git",
  "--help",
  "--log_file",
  "--output_dir",
  "--overwrite",
  "--post_work_executable",
  "--save_config",
  "--background",
  "--debug",
  "--json",
  "--licfile",
  "--maxlim",
  "--mpi_exec_path",
  "--nm_version",
  "--nmqual",
  "--nobuild",
  "--parafile",
  "--parallel",
  "--parallel_timeout",
  "--prcompile",
  "--preview",
  "--prsame",
  "--threads",
  "--verbose"
)

f3 <- c(
  "--ext-file",
  "--help",
  "--no-ext-file",
  "--no-grd-file",
  "--no-shk-file",
  "--background",
  "--debug",
  "--json",
  "--licfile",
  "--maxlim",
  "--mpi_exec_path",
  "--nm_version",
  "--nmqual",
  "--nobuild",
  "--parafile",
  "--parallel",
  "--parallel_timeout",
  "--prcompile",
  "--preview",
  "--prsame",
  "--threads",
  "--verbose"
)


new_flags <- sort(unique(c(f1, f2, f3)))

old_flags <- map_chr(NONMEM_ARGS, "flag")

setdiff(new_flags, old_flags)
# [1] "--additional_post_work_envs" "--background"                "--help"                      "--licfile"                  
# [5] "--log_file"                  "--maxlim"                    "--nobuild"                   "--parallel_timeout"         
# [9] "--post_work_executable"      "--prcompile"                 "--prsame"  

setdiff(old_flags, new_flags)
# [1] "--cache_dir"    "--cache_exe"    "--gitignoreLvl" "--no-cor-file"  "--no-cov-file"  "--nodes"        "--save_exe"     "--timeout" 
```

**Comment by copernican** [comment]
As mentioned in #126, would we consider not checking, or maintaining a list of, arguments in `rbabylon` and bubbling up errors from `bbi` instead?

**Comment by seth127** [comment]
removing because not needed for validation

**Comment by seth127** [comment]
Opened a new issue for potential next steps: https://github.com/metrumresearchgroup/rbabylon/issues/171
